docs/tools/piral-inspector.md

```diff
@@ -2,229 +2,199 @@
 
-![Available pilets from the Piral Inspector](../diagrams/piralinspector_tool_availablepilets.png)
+![Dependency Graph](../diagrams/piralinspector-feature-dependencies-graph.png)
 
-By clicking on the switch button on the left, the user can activate and deactivate each pilet.
+## Settings
 
-With a click on the "X" on the right, the user can remove each pilet individually.
+::: tip: TL;DR
+Both the extension and the debugged application can be configured to some degree. Some settings only work when debugging an _app shell_, while others only work when debugging _pilets_.
+:::
 
-### Add Pilets
+Piral Inspector provides several settings, both for the extension itself and the debugged application. The settings have the following effects:
 
-There are two options to load additional pilets into the currently running Piral Instance:
+### State Container Logging
 
-#### Add a Feed Address
+Enables/Disables the state container console logs which are, by default, emitted by Piral. You can disable this if your browser console feels cluttered.
 
-Enter a feed address into the text field and click on the "Add" button.
+![State Container Logs](../diagrams/piralinspector-setting-state-container-logging.png)
 
-In case you chose a feed you have access to, the corresponding pilets will be loaded and shown in the tile section and the list of available pilets. Per default, the new pilets will be active.
+### Load Available Pilets
 
-#### Bundle Local Pilet and Upload It
+::: warning: Only for pilets
+This feature only has an impact when debugging _pilets_, i.e., when `pilet debug` is running.
+:::
 
-First, you need to "Browse" to the pilet package file and then "Upload" it.
+Activates/Deactivates the loading of the available pilets.
 
-In case you uploaded a proper pilet, it will be loaded and shown in the tile section and the list of available pilets. By default, the new pilet will be active but will be given a random name.
+### Full refresh on change
 
-::: warning: Temporary feed
-The pilet will be loaded into a temporary feed, and this feed (including the pilets) will be removed when the predefined lifetime (by default 60 minutes) expires.
+::: warning: Only for pilets
+This feature only has an impact when debugging _pilets_, i.e., when `pilet debug` is running.
 :::
 
-![Add pilets to the Piral Inspector](../diagrams/piralinspector_tool_addpilets.png)
-
-### Registered Routes
+Refreshes the current browser page when the currently debugged pilet changes.
 
-This section shows all currently registered routes, and with a click on a single one, it will call the route.
+### Visualize component origins
```

Comment:
Can we keep title case?

Above we have things like "State Container Logging" but then we have "Full refresh on change" or "Visualize component origins". Would be great to use instead "Full Refresh on Change" or "Visualize Component Origins" etc.

Reply:
I can change that, but in this specific case I did it on purpose (I copied the name of the setting 1:1).